templates/global/roll-prompt.hbs

```diff
@@ -28,7 +28,7 @@
 
 
                     <!-- 30/20 -->
-                    {{#if ( uneq rollType CONFIG.ED4E.rollTypes.arbitrary) }}
+                    {{#if ( uneq options.rollType arbitrary) }}
```

Comment:
If I roll from the char sheet I don't see the strain and modifier options anymore.
This should be just `uneq rollType arbitrary`.  The rollOptions are unpacked in the context data, so they can just be accessed by name, without qualifier in front of it.

Reply:
ok check. solved it